memgraph_versioned_docs/version-2.3.1/connect-to-memgraph/drivers/php.md

```diff
@@ -89,8 +88,14 @@ You should see an output similar to the following:
 Node 1: Hello, World!
 ```
 
+_If you run apache/nginx instance you can open your project in web browser and you will get same result._
+
+
+
 ## Where to next?
 
+To learn more about using PHP Bolt libary take a look at readme and wiki at https://github.com/neo4j-php/Bolt.
```

Comment:
Check out the [Bolt driver repository](https://github.com/neo4j-php/Bolt) to learn more about using the PHP Bolt library.